src/components/apis/list-of-apis/ko/listOfApisViewModelBinder.ts

```diff
@@ -30,11 +30,16 @@ export class ListOfApisViewModelBinder implements ViewModelBinder<ListOfApisMode
                 : undefined
         }));
 
+        const handler = model.layout == "list" ? ListOfApisHandlers :
+            model.layout == "dropdown" ?
+                ListOfApisDropdownHandlers :
+                ListOfApisTilesHandlers;
```

Comment:
Threat this comment as nit-pick and a great opportunity to learn.

In the next assignment you need to add a next type "popup" of handlers. Will you add a next case to that ternary operator?

```js
model.layout == "list" 
? ListOfApisHandlers 
: model.layout == "dropdown" 
?  ListOfApisDropdownHandlers 
: model.layout == "popup" 
? ListOfPopupHandlers
: ListOfApisTilesHandlers;
``` 
I think it is quite complex and had to maintain. Instead of that you can use object.

 ```js
const handlerForLayout = {
  list: ListOfApisHandlers,
  dropdown: ListOfApisDropdownHandlers,
  popup: ListOfPopupHandlers,
};
const handler = handlerForLayout[model.layout] ?? ListOfApisTilesHandlers;
```

I hope it will help you :)

Reply:
You are right. It is clearer like this and easier to maintain. Thanks for pointing it out.